io.github.mudkipworld.pngtuber-remix.yml

```diff
@@ -0,0 +1,127 @@
+id: io.github.mudkipworld.pngtuber-remix
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+base: org.godotengine.godot.BaseApp
+base-version: '4.4-25.08'
+sdk: org.freedesktop.Sdk
+command: pngtuber-remix
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  # App requires network access for it's protocol to contro the puppet remotely
+  - --share=network
+
+modules:
+  - name: scons
+    buildsystem: simple
+    sources:
+      # Source0
+      - type: archive
+        url: https://flylife.dl.sourceforge.net/project/scons/scons/4.10.1/SCons-4.10.1.tar.gz
+        sha256: 99c0e94a42a2c1182fa6859b0be697953db07ba936ecc9817ae0d218ced20b15
+    build-commands:
+      - python3 setup.py install --prefix="${FLATPAK_DEST}";
+    cleanup:
+      - '*'
+
+  - name: pngtuber-remix
+    buildsystem: simple
+
+    sources:
+      - type: git
+        url: https://github.com/MudkipWorld/PNGTuber-Remix.git
+        tag: V1.4.1
+        disable-shallow-clone: true
+      - type: git
+        url: https://github.com/MudkipWorld/Godot-Global-Input.git
+        commit: ace1775114330f925579604e2459649464a38306
+        dest: ./.deps/GI
+      - type: git
+        url: https://github.com/BOTLANNER/godot-gif.git
+        tag: 1.1.1
+        dest: ./.deps/godot-gif
+      - type: git
+        url: https://github.com/MudkipWorld/GDExtensionsPlayGround.git
+        commit: 5ea7f79120ed12f145b4bafe956ced2e10cbc8c3
+        dest: ./.deps/GDExtensionsPlayGround
+      - type: patch
```

Comment:
why so many patches from the get go?

Reply:
The first 2 are the only ones needed. Since the upstream app is intended to be used as a standalone binary, config files are stored besides the binary by default, this patches that to use the default config path.

The rest are needed to recompile the bundled libraries so that they can be used in aarch64, as requested by the other reviewer.